### DISCUSSION TITLE
Demo of how to run traefik with socket activation + rootless Podman + Quadlet files

### DISCUSSION THREAD

**Discussion by eriksjolund** [containers/podman#23076] [Show and tell]
I tried out this __traefik/traefik__ PR with rootless Podman and Quadlet files

* https://github.com/traefik/traefik/pull/10399

A minimal example where a socket-activated traefik container acts as an HTTP reverse proxy works just fine. 
For details, see
https://github.com/eriksjolund/podman-traefik-socket-activation

The traefik project is [planning](https://github.com/traefik/traefik/pull/10399#issuecomment-2133376924) to add _socket activation_ support in traefik v3.1.0 (which has not yet been released).

**Comment by ygalblum** [comment]
@eriksjolund Thanks for sharing. Would you like to contribute this to our demos repository: https://github.com/containers/appstore

**Comment by eriksjolund** [comment]
In the current state I think the demo is a bit	too bleeding edge to be included	in
https://github.com/containers/appstore
It would be good to wait until _socket activation_ support is released in an official version of traefik.
(Or at least that we could find a semi-official container image with  _socket activation_ support)

Another idea is to improve the official traefik documentation. For example, a quadlet example
would be good to have next to the docker-compose example:
https://doc.traefik.io/traefik/user-guides/docker-compose/basic-example/
but probably _socket activation_ support needs to land first.
